Sources/ApodiniHTTP/InterfaceExporter.swift

```diff
@@ -58,6 +63,35 @@ public final class HTTP: Configuration {
         
         /// Insert exporter into `InterfaceExporterStorage`
         app.registerExporter(exporter: exporter)
+        
+        self.staticConfigurations.configureAny(app, parentConfiguration: self.configuration)
+    }
+}
+
+extension HTTP {
+    /// Initializes the configuration of the `HTTPInterfaceExporter` with (default) JSON Coders and possibly associated configurations, e.g. OpenAPI or APIAuditor
+    /// - Parameters:
+    ///    - encoder: The to be used `JSONEncoder`, defaults to a `JSONEncoder`
+    ///    - decoder: The to be used `JSONDecoder`, defaults to a `JSONDecoder`
+    ///    - caseInsensitiveRouting: Indicates whether the HTTP route is interpreted case-sensitively
+    ///    - rootPath: Configures the root path for the HTTP endpoints
+    ///    - staticConfigurations: A result builder that allows passing dependent static Exporters like the OpenAPI Exporter
+    public convenience init(
+        encoder: JSONEncoder = defaultEncoder as! JSONEncoder,
+        decoder: JSONDecoder = defaultDecoder as! JSONDecoder,
+        urlParamDateDecodingStrategy: ApodiniNetworking.DateDecodingStrategy = .default,
+        caseInsensitiveRouting: Bool = false,
+        rootPath: RootPath? = nil,
+        @DependentStaticConfigurationBuilder<HTTP> staticConfigurations: () -> [AnyDependentStaticConfiguration] = { [] }
```

Comment:
As noted above, as far as I can see, the constraint currently doesn't have any effect here.